################################################################################
SHORT DESCRIPTION: 
################################################################################
Use the first fb driver loaded instead of the primary device while using DRM
KMS helper.

################################################################################
LONG DESCRIPTION: 
################################################################################
Remove auto-select FRAMEBUFFER_CONSOLE_DETECT_PRIMARY from DRM_KMS_FB_HELPER.

################################################################################
CHANGELOG 
################################################################################
Original Author: unknown
Port to 3.18: Eric Chanudet <chanudete@ainfosec.com>

################################################################################
REMOVAL 
################################################################################

################################################################################
UPSTREAM PLAN
################################################################################
None, this is an OpenXT work-around.

################################################################################
INTERNAL DEPENDENCIES 
################################################################################
Is required by surfman/drm-plugin.

################################################################################
PATCHES 
################################################################################
Index: linux-3.18.24/drivers/gpu/drm/Kconfig
===================================================================
--- linux-3.18.24.orig/drivers/gpu/drm/Kconfig	2015-11-05 04:18:25.393613065 +0100
+++ linux-3.18.24/drivers/gpu/drm/Kconfig	2015-11-05 04:18:35.613475516 +0100
@@ -36,7 +36,6 @@
 	depends on DRM_KMS_HELPER
 	select FB
 	select FRAMEBUFFER_CONSOLE if !EXPERT
-	select FRAMEBUFFER_CONSOLE_DETECT_PRIMARY if FRAMEBUFFER_CONSOLE
 	help
 	  FBDEV helpers for KMS drivers.
 
